### PR TITLE
Add testing for FindTagsCommand and parser

### LINKED ISSUE #201: findTags command and parser testing

### PR DESCRIPTION
**Pull request by NecroWolf28**
Fixes #201

### CODE REVIEW

File: src/test/java/seedu/placebook/logic/commands/FindTagsCommandTest.java [hunk 1]
```diff
@@ -67,6 +67,26 @@ public void execute_zeroKeywords_noPersonFound() {
...
+        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 0);
+        PersonHasTagsPredicate predicate = preparePredicate("enemies");
+        FindTagsCommand command = new FindTagsCommand(predicate);
+        expectedModel.updateFilteredPersonList(predicate);
+        assertCommandSuccess(command, model, uiStub, expectedMessage, expectedModel);
+        assertEquals(Collections.emptyList(), model.getFilteredPersonList());
+    }
+
+    @Test
+    public void execute_oneKeyword_onePersonFound() {
```
Comment (nwhloo99) [inline src/test/java/seedu/placebook/logic/commands/FindTagsCommandTest.java:81]: I think can add few test case:  
one keyword and multiplePersonFound
multipleKeywords and onePerson

**Review by nwhloo99 — COMMENTED** [review]
Should be more or less done, just maybe need add a few test case

File: src/test/java/seedu/placebook/logic/commands/FindTagsCommandTest.java (hunk 1 above)
Comment (NecroWolf28) [reply src/test/java/seedu/placebook/logic/commands/FindTagsCommandTest.java:81]: alrighty

**Review by YaleChen299 — APPROVED** [review]
LGTM

**Pull request merged**